user-service/pom.xml

```diff
@@ -44,6 +45,20 @@
             <groupId>org.java.mentorship</groupId>
             <artifactId>user-contracts</artifactId>
         </dependency>
+
+
+        <dependency>
+            <groupId>org.mybatis.spring.boot</groupId>
+            <artifactId>mybatis-spring-boot-starter-test</artifactId>
+            <version>${mybatis.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>${h2.version}</version>
```

Comment:
We can probably move the version of these dependency to the parent's dependencyManagement, since all modules will probably need the h2 dependency